### ISSUE TITLE
Model not performing well on validation data

### ISSUE THREAD

**Issue by dhruvsharma15** [Issam28/Brain-tumor-segmentation#22]
I am trying to run this model on Brats 2018 dataset. To evaluate my model, I'm using a portion of the same dataset as validation. But, the model yields very bad values for the validation split. Here are the stats after 40 epochs.
Training loss = 0.56
dice_whole_metric = 0.81
dice_en_metric = 0.62

Validation loss = 9.60
dice_whole_metric = 0.0025
dice_en_metric = 4064e-06

What possible reasons/solutions are there for this issue?
